### ISSUE TITLE
Metrics do not display zero-values correctly

### ISSUE THREAD

**Issue by echlebek** [sensu/sensu-go#1460]
## Expected Behavior
I expect to see
```
      "points": [
        {
          "name": "percent_packet_loss",
          "value": 0,
          "timestamp": 1525723224,
          "tags": []
        },
        {
          "name": "rta",
          "value": 0.8,
          "timestamp": 1525723224,
          "tags": []
        }
      ]
```

## Current Behavior
I see:
```
      "points": [
        {
          "name": "percent_packet_loss",
          "timestamp": 1525723224,
          "tags": []
        },
        {
          "name": "rta",
          "value": 0.8,
          "timestamp": 1525723224,
          "tags": []
        }
      ]
```

## Possible Solution
Remove `omitempty` json tag from the generated datatype.

## Steps to Reproduce (for bugs)
1. create a check that emits `PING ok - Packet loss = 0%, RTA = 0.80 ms | percent_packet_loss=0, rta=0.80` with a metric format of `nagios_perfdata`.
2. List the generated event in JSON format.

## Context
Bug filed while doing beta QA.

## Your Environment

* Sensu version used: e91bfb3194d6be72e1b74a7a9500afe0a765f0c7
* Build and run method: `./build.sh docker`, `docker-compose up`
* Operating System and version: Fedora 26
